src/index.ts

```diff
@@ -6,20 +6,10 @@ import { app } from './infrastructure/api';
 
 import { createMajorVersionNumber, createHandlerBasePath } from './utils';
 
-const {
-  NODE_ENV, API_VERSION, AWS_PROVIDER_REGION, AWS_PROVIDER_STAGE,
-} = process.env;
+const { API_VERSION } = process.env;
```

Comment:
Not an issue, but just curious how this statement works. Intuitively, when reading this I assumed this meant `process.env` would be copied over into a new full object - instead, is this actually slicing the `env` into just the one field?

Reply:
Yeah, it's object destructuring, so it pulls out the value associated with the key that you use as a valirable name e.g.

```
const myObject = {
    keyA: "test",
    keyB: "dave"
}

const {keyA, keyB} = myObject

console.log(keyA) // "test"
console.log(keyB) // "dave"
```